applications/FluidDynamicsApplication/python_scripts/symbolic_generation/compressible_navier_stokes/src/defines.py

```diff
@@ -0,0 +1,17 @@
+import sympy
```

Comment:
Maybe. There are functions with the same name in the core but generate diferent outputs. 
```
DefineMatrix('A', numrows, numcols)[0,3]:
core -> A_0_3
mine -> A(0,3)   {and assumed to be a real number}
```
These are somewhat more specific so I decided to create them anew to avoid braking other symbolic scripts.

Reply:
Added the functions as static of a class to avoid colisions with core's version of these function. We can add them to the core if we see it fit in a future PR.